media-api/app/lib/ImageResponse.scala

```diff
@@ -230,8 +231,18 @@ object ImageResponse extends EditsResponse {
     (__ \ "usages").write[UsagesEntity]
       .contramap(usagesEntity(id, _: List[Usage])) ~
     (__ \ "collections").write[List[EmbeddedEntity[Collection]]]
-      .contramap((collections: List[Collection]) => collections.map(c => collectionsEntity(id, c)))
-  )(unlift(Image.unapply))
+      .contramap((collections: List[Collection]) => collections.map(c => collectionsEntity(id, c))) ~
+    (__ \ "collectionGroups").write[List[CollectionGroup]]
```

Comment:
Just spoke to @tsop14 - it's be good to have a chat, as, as far as I was aware, the most important thing from a users perspective was to know that another group was using the image, not really what leaf it was in as filtering, rather than scanning, has become ichiban.